frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -103,11 +100,34 @@ const ConsolePluginEnabledStatus: React.FC<ConsolePluginEnabledStatusProps> = ({
     name: CONSOLE_OPERATOR_CONFIG_NAME,
   });
 
+  return React.useMemo(
+    () => ({ consoleOperatorConfig, consoleOperatorConfigLoaded, canPatchConsoleOperatorConfig }),
+    [consoleOperatorConfig, consoleOperatorConfigLoaded, canPatchConsoleOperatorConfig],
+  );
+};
+
+export const ConsolePluginStatus: React.FC<ConsolePluginStatusProps> = ({
+  status,
+  errorMessage,
+}) => <Status status={status} title={status === 'Failed' ? errorMessage : undefined} />;
+
+export const ConsolePluginEnabledStatus: React.FC<ConsolePluginEnabledStatusProps> = ({
+  pluginName,
+  enabled,
+}) => {
+  const { t } = useTranslation();
+
+  const {
+    consoleOperatorConfig,
+    consoleOperatorConfigLoaded,
+    canPatchConsoleOperatorConfig,
+  } = useConsoleOperatorConfigData();
+
   const labels = enabled ? t('console-app~Enabled') : t('console-app~Disabled');
 
   return (
     <>
-      {consoleOperatorConfig && canPatchConsoleOperatorConfig ? (
+      {consoleOperatorConfigLoaded && canPatchConsoleOperatorConfig && !developmentMode ? (
```

Comment:
Fixes regression introduced in https://github.com/openshift/console/pull/14403.  We only want to display the button when not in dev mode.